packages/core/src/domain/session/sessionManagement.ts

```diff
@@ -24,14 +33,34 @@ export function startSessionManagement<TrackingType extends string>(
   const sessionStore = startSessionStore(options, productKey, computeSessionState)
   stopCallbacks.push(() => sessionStore.stop())
 
+  const sessionContextHistory = new ContextHistory<SessionContext<TrackingType>>(SESSION_CONTEXT_TIMEOUT_DELAY)
```

Comment:
Since we are using ContextHistory here now, wouldn't it make sens to rename `sessionManagment` to `sessionContexts`. We would be consistent with `urlContexts`, `parentContexts` and we could have the API looking like `sessionContexts.findSession(startTime)`

Reply:
I would say no.
`urlContext` and `parentContexts` only maintain the contexts, `sessionManagement` has more responsibilities.
About `findSession(startTime)`, it seems to be the same idea as https://github.com/DataDog/browser-sdk/pull/1187#issuecomment-983723926, no?

Reply:
Let's discuss the naming when doing [ #1187 (comment)](https://github.com/DataDog/browser-sdk/pull/1187#issuecomment-983723926)